pom.xml

```diff
@@ -51,6 +51,7 @@
         <maven.build.timestamp.format>yyyyMMdd.HHmmss</maven.build.timestamp.format>
         <javafx.version>19.0.2.1</javafx.version>
         <maven.compiler.release>11</maven.compiler.release>
+        <project.build.outputTimestamp>2023-01-01T00:00:00Z</project.build.outputTimestamp>
```

Comment:
It's fine. In future maybe it's better to use git commits? Like article says:
"instead of explicitely writing a timestamp in their pom.xml, some people tend to prefer using last Git commit timestamp, like ${git.commit.time} from [git-commit-id-maven-plugin](https://github.com/git-commit-id/git-commit-id-maven-plugin)."

Anyway, it's fine to leave as is if you think it's needed